harbors/management/commands/update_harbors_from_servicemap.py

```diff
@@ -0,0 +1,54 @@
+import requests
+
+from django.contrib.gis.geos import Point
+from django.conf import settings
+from django.core.management.base import BaseCommand
+
+from munigeo.models import Municipality
+
+from harbors.models import Harbor
+
+
+class Command(BaseCommand):
+
+    @staticmethod
+    def _get_servicemap_url(sm_id):
+        return 'https://api.hel.fi/servicemap/v2/unit/{}/'.format(sm_id)
+
+    def handle(self, **options):
+        updated_harbors = 0
+
+        for harbor in Harbor.objects.all():
+            self.stdout.write(
+                'Fetching info for harbor with servicemap ID {}'.format(harbor.servicemap_id)
+            )
+
+            sm_response = requests.get(self._get_servicemap_url(harbor.servicemap_id))
+            sm_info = sm_response.json()
+
+            harbor.zip_code = sm_info['address_zip']
+            harbor.www_url = sm_info['www']['fi']  # only FI for now
+            harbor.email = sm_info['email']
+            harbor.phone = sm_info['phone']
+
+            harbor.municipality = Municipality.objects.get(id=sm_info['municipality'])
+
+            latitude = sm_info['location']['coordinates'][1]
+            longitude = sm_info['location']['coordinates'][0]
+            harbor.location = Point(longitude, latitude, srid=settings.DEFAULT_SRID)
+
+            harbor.save()
```

Comment:
Is there some reason why there cannot be just one `harbor.save()` at the end of the loop?

Reply:
Not sure, that you can set properly translated fields without saving each language... This `harbor.save()` line might be deleted though? if we save the object pretty soon when handling translated field?

Reply:
Ok, sounds a bit stupid, but might be, I don't know Parler that well :) If that is the case, I would keep this save as well to make sure the object is saved even when there are no translations in the upcoming translated fields (probably not possible in practice but still).